src/devices/README.md

```diff
@@ -227,34 +243,44 @@ Our vision: the majority of .NET bindings are written completely in .NET languag
 
 ### USB devices
 
+* [Board abstraction](Board/README.md)
+* [SPI, GPIO and I2C drivers for Arduino with Firmata](Arduino/README.md)
 * [SPI, GPIO and I2C drivers for FT4222](Ft4222/README.md)
 * [STUSB4500 - Autonomous USB-C PD controller for Power Sinks / UFP](StUsb4500/README.md)
 
 ### GPIO or bit operating devices
 
+* [Board abstraction](Board/README.md)
 * [Charlieplex Segment binding](Charlieplex/README.md)
 * [Digital liquid level switch](LiquidLevel/README.md)
 * [Generic shift register](ShiftRegister/README.md)
+* [GpioDriver for other boards](Gpio/README.md)
 * [Key Matrix](KeyMatrix/README.md)
 * [MBI5027 -- 16-bit shift register with error detection](Mbi5027/README.md)
 * [Quadrature Rotary Encoder](RotaryEncoder/README.md)
 * [SN74HC595 -- 8-bit shift register](Sn74hc595/README.md)
+* [SPI, GPIO and I2C drivers for Arduino with Firmata](Arduino/README.md)
 
 ### Multi-device or robot kit
 
 * [GoPiGo3](GoPiGo3/README.md)
 * [GrovePi](GrovePi/README.md)
 * [Sense HAT](SenseHat/README.md)
+* [SensorHub - Environmental sensor](SensorHub/README.md)
```

Comment:
perhaps arduino and board should be just here and under protocols

Reply:
this one is fine, it's used with multi devices like the previous ones